constraint/pkg/client/regolib/src.go

```diff
@@ -7,11 +7,15 @@ package hooks["{{.Target}}"]
 violation[response] {
 	data.hooks["{{.Target}}"].library.autoreject_review[rejection]
 	review := get_default(input, "review", {})
+	constraint := get_default(rejection, "constraint", {})
+	spec := get_default(constraint, "spec", {})
+	enforcementaction := get_default(spec, "enforcementaction", "deny")
```

Comment:
"deny" or "DENY"?

Reply:
so are we requiring all `enforcementAction` to be cap?

Reply:
ya, it's golang and k8s api convention to use camel case

Reply:
My thought is that traditionally keywords like this have been cap, so I thought it might make sense to continue the tradition. (ex ip tables)

Curious if anyone else has any opinion.